package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "paper-password-input",
+  "authors": [
+    "collaborne"
+  ],
+  
+  "description": "A Material Design input field to enter passwords (Polymer)",
+  "main": "paper-password-input.html",
+  "keywords": [
+    "web-components",
+    "polymer",
+    "paper",
+    "material design",
+    "input",
+    "password"
+  ],
+  "_requested": {
```

Comment:
This `_requested` block looks wrong, accidentally left-over from some automatic tool?